content/2017-munich/sponsor.md

```diff
@@ -69,12 +69,12 @@ prices and benefits:
     </tr>
     <tr>
       <td>
-        Logo on all videos after the event
+        Logo on onsite signage (off-stage)
```

Comment:
I'd remove it from both.

There should be such a place if we can fit it, but best not to guarantee it.